lib/fastlane/plugin/wpmreleasetoolkit/actions/common/create_release_backmerge_pull_request_action.rb

```diff
@@ -0,0 +1,168 @@
+require 'fastlane/action'
+require_relative '../../helper/github_helper'
+
+module Fastlane
+  module Actions
+    class CreateReleaseBackmergePullRequestAction < Action
+      DEFAULT_BRANCH = 'trunk'.freeze
+
+      def self.run(params)
+        token = params[:github_token]
+        repository = params[:repository]
+        source_branch = params[:source_branch]
+        default_branch = params[:default_branch]
+        target_branches = params[:target_branches]
+        labels = params[:labels]
+        milestone_title = params[:milestone_title]
+        reviewers = params[:reviewers]
+        team_reviewers = params[:team_reviewers]
+
+        if target_branches.include?(source_branch)
+          UI.user_error!('`target_branches` must not contain `source_branch`')
+        end
+
+        github_helper = Fastlane::Helper::GithubHelper.new(github_token: params[:github_token])
+        target_milestone = milestone_title.nil? ? nil : github_helper.get_milestone(repository, milestone_title)
+
+        final_target_branches = if target_branches.empty?
+                                  determine_target_branches(release_version: source_branch.delete('release/'), default_branch: default_branch)
```

Comment:
We'd still want to keep your check that the `source_branch` starts with `release/` in the case where `target_branches.empty?`—since `determine_target_branches(release_version:, …)` expects the `release_version` parameter to be string representing a version number for its implementation logic to work.

_(It's only when `target_branches` is non-empty that we want to allow non-`release/`-prefixed `source_branch` to be ok.)_